src/ofxGLWarper.cpp

```diff
@@ -288,8 +287,8 @@ void ofxGLWarper::mousePressed(ofMouseEventArgs &args){
 
     cornerSelected = false;
 	for(int i = 0; i < 4; i++){
-		float distx = corners[i].x - (float)args.x;
-		float disty = corners[i].y - (float)args.y;
+        float distx = corners[i].x - static_cast<float>(args.x);
```

Comment:
I can't tell why args.x was being cast as it is actually a float. Instead of adding the static_cast I'd rather remove it all. can you make this change please?